fabfile/fabric_settings.py.default.py

```diff
@@ -1,39 +1,51 @@
+"""
+Welcome to the `fabric_settings.py` file!
+
+Below is a field that will fill in the Ansible configurations with your desired
+values. Once you're finished, save and then do a fab command. To see the list of
+fab commands, enter in `fab -l`.
+
+If you're filling this out for an existing Yurt project, the only necessary fields
+to fill in are `git_repo` and `project_name`. The other options are only necessary
```

Comment:
I jumped the gun, looks like you're having them enter these two fields for existing projects. Would it be more approachable to have someone enter them when they run the fab command?